### ISSUE TITLE
How to use in react project

### ISSUE THREAD

**Issue by SaeedZhiany** [google/wicked-good-xpath#57]
Is it possible to use this library in react projects?

**Comment by Dominator008** [comment]
Should be. `wgxpath` is exported as a global on `window`.

**Comment by SaeedZhiany** [comment]
Is there any way in a more react-like way?
I mean something like below:

```
import wgxpath from 'wicked-good-xpath';

wgxpath.evaluate(/*Some xpath*/);
```

**Comment by Dominator008** [comment]
Unfortunately that's not supported. This library pre-dates ES2015 or even CommonJS modules :P

**Comment by SaeedZhiany** [comment]
Is there any chance to support it in near future? is this repository no longer maintained?

**Comment by SaeedZhiany** [comment]
Is there any chance to support it in near future? or this repository is no longer maintained?

**Comment by Dominator008** [comment]
We are not adding new functionalities or changing how it's exported. I think you can still add it to your html and access `wgxpath` from `window`, right?

**Comment by SaeedZhiany** [comment]
yeah, it should be possible, but it's not a normal way in react projects.
